src/OrleansServiceFabricUtils/OrleansCommunicationListener.cs

```diff
@@ -42,7 +42,8 @@ public class OrleansCommunicationListener : ICommunicationListener
         /// </summary>
         /// <param name="context">The context.</param>
         /// <param name="configuration">The configuration.</param>
-        public OrleansCommunicationListener(ServiceContext context, ClusterConfiguration configuration)
+        /// <param name="siloName">The silo name, or <see langword="null"/> to have a name automatically selected.</param>
+        public OrleansCommunicationListener(ServiceContext context, ClusterConfiguration configuration, string siloName = null)
```

Comment:
Just curious. The siloName seems to be different than siloName defined in ClusterConfiguration. Why is that?